app/read_models/caseworker_reports/projection.rb

```diff
@@ -0,0 +1,60 @@
+module CaseworkerReports
+  class Projection
```

Comment:
I'm not sure what projection is meant to mean?

Reply:
https://www.eventstore.com/event-sourcing#Projections